jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/HoverTests.java

```diff
@@ -17,70 +17,103 @@
-                    Matchers.hasItem(subheading)
-            )
-        );
+    @Test(description = "Assert that hover effect is enabled and present only when card is being hovered")
+    public void hoverEffectIsPresentTest() {
+        Stream.of(openDelayHover, closeDelayHover).forEach(item -> {
+            item.has().cssClass("elevation-2");
+            item.show();
+            item.has().cssClass("elevation-16");
+            item.has().cssClass("on-hover");
+            item.suppress();
+            item.has().cssClass("elevation-2");
+        });
+
+        musicHoverCards.forEach(item -> {
+            item.has().cssClass("elevation-2");
+            item.getHover().show();
+            item.has().cssClass("elevation-12");
+            item.has().cssClass("on-hover");
+            item.getHover().suppress();
+            item.has().cssClass("elevation-2");
+        });
+    }
 
+    @Test(description = "Assert that Music Card list player buttons appear only when card is being hovered")
+    public void musicCardListPlayerButtonsAppearWhenHoveredTest() {
         musicHoverCards.forEach(card -> {
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
 
             card.getHover().show();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert.cssClass("show-btns"));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClass("show-btns"));
+
+            card.getPlayerButtons().forEach(button -> button.core().isClickable());
             card.getPlayerButtons().forEach(HasClick::click);
 
             card.getHover().suppress();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
         });
     }
 
-    @Test
-    public void cookingCardTest() {
-        cookingHeader.has().text(containsString("cooking utensils"));
-        cartButton.click();
+    @Test(description = "Assert that Music Card list has proper texts")
+    public void musicCardListTextsTest() {
+        SUBHEADINGS.forEach(subheading -> jdiAssert(musicHoverCards.stream()
+                .map(MusicHoverCard::getSubheading)
+                .map(IsText::getText)
+                .collect(Collectors.toList()), Matchers.hasItem(subheading)));
+    }
+
```

Comment:
Hi :) Is there possibility to check: one item of list is hovered, others are not?